.github/workflows/iroha2-dev-lints.yml

```diff
@@ -0,0 +1,111 @@
+name: I2::Dev::Lints
```

Comment:
I don't think the name "lints" is representative of the jobs

Reply:
I agree, let's propose something appreciable

Reply:
Hmm, how about `CodeQuality`, which seems to be one of the pronouns of SonarQube. But this may not make it clear that tests are included

Reply:
```suggestion
name: I2::Dev::CodeQuality
```